package.json

```diff
@@ -37,7 +37,7 @@
     "@alfresco/adf-core": ">=6.6.0-7017572520",
     "@alfresco/adf-extensions": ">=6.6.0-7017572520",
     "@alfresco/eslint-plugin-eslint-angular": ">=6.6.0-7017572520",
```

Comment:
What about the adf version? Should we bump it as well?

Reply:
adf has: 
>=6.6.0-7017572520
so I think it's fine?

Reply:
That depends on the version you want to use later on in ADW, ideally they should be the same to mitigate the risk of NPM conflicts during the install

Reply:
Good practice is to use latest ADF alpha in both ACA and ADW